src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -253,11 +253,20 @@ int net_device_table_mgr::map_net_devices()
 			continue;
 		}
 
+		int iftype = get_iftype_from_ifname(ifa->ifa_name);
+		if (iftype == ARPHRD_INFINIBAND) {
+			if (!check_ib_device_supported(ifa->ifa_name) || !verify_ipoib_mode(ifa)) {
+				continue;
+			}
+		} else {
+			if (!check_qp_creation(ifa->ifa_name, IBV_QPT_RAW_PACKET)) {
```

Comment:
2. so used can learn QP creation fail, but we don't give any hint on how to fit this..... running as root or enabling CAP_NET_RAW permissions
if QP create fails we should also check if user has CAP_NET_RAW  and if not print some hint in INFO on how to 'fix' this, because this is fixable